src/node_file.cc

```diff
@@ -1322,8 +1322,8 @@ int MKDirpAsync(uv_loop_t* loop,
           std::string dirname = path.substr(0,
                                             path.find_last_of(kPathSeparator));
           if (dirname != path) {
-            req_wrap->continuation_data->PushPath(std::move(path));
-            req_wrap->continuation_data->PushPath(std::move(dirname));
+            req_wrap->continuation_data->PushPath(path);
+            req_wrap->continuation_data->PushPath(dirname);
```

Comment:
This adds copy operations at every call site? I don’t think that’s what you’re going for, right?